cli/seedAccessControl/grants/Hosts.js

```diff
@@ -15,9 +15,9 @@ const Users = [
     attributes: '*',
   },
 
-  // Show Producer
+  // Show Captain
   {
-    role: 'Show Producer',
+    role: 'Show Captain',
```

Comment:
@dzimmerman902 I updated the role name here: do you know if there's any test data that needs to be changed for this?

Reply:
@seankwilliams Not that I can think of, should be good!